### PR TITLE
Improve many aspects of the coordinator interface

### PR DESCRIPTION
**Pull request by thecalliopecrashed**
* Inline the `Callout` component - it's small enough that it makes sense in different locations.
* Bring back the icons that @campbellsean introduced in #71.
* Decide on how the status icons work with @sjiribarren - There's an additional commit on its way.

### CODE REVIEW

**Comment by thecalliopecrashed** [comment]
Screen Shot 2019-03-27 at 3 31 49 PM

File: bin/deploy [hunk 1]
```diff
@@ -24,10 +24,11 @@ sudo mv build /srv/www/tb-app.cirg.washington.edu/htdocs
 # sudo docker-compose exec web bundle exec rake db:create db:migrate
 
 # # Soft Restart!
-sudo docker-compose kill -f web client
+sudo docker-compose stop
 sudo docker-compose rm -f web client
 sudo docker-compose pull
 sudo docker-compose up -d
+sudo docker-compose exec web bundle exec rake db:create db:migrate
```
Comment (thecalliopecrashed) [inline bin/deploy]: A couple of tweaks to the soft restart process.

File: models/coordinator.rb [hunk 2]
```diff
@@ -11,6 +11,7 @@ def as_json(*args)
       name: name,
       email: email,
       participants: Participant.all.as_json,
+      resolutions: Resolution.all.as_json,
```
Comment (thecalliopecrashed) [inline models/coordinator.rb]: One step closer to "resolving" reports.

File: bin/deploy (hunk 1 above)
Comment (campbellsean) [reply bin/deploy]: Nice, so we make any pending migrations for each soft restart

File: src/components/CoordinatorHome.js [hunk 3]
```diff
@@ -154,24 +200,42 @@ const InlineButton = styled(Button)`
...
-//   return unique_report_dates.length / full_days
-// }
+const Cell = styled.div`
+  padding: 0.5rem;
+`
+
+// Need a thorough review of this logic;
+// it seems to work for now,
+// but won't hold up in the live site.
+const participant_has_reports_that_have_not_been_resolved = (participant) => (
```
Comment (campbellsean) [inline src/components/CoordinatorHome.js]: Nice

**Pull request closed without merging**